pkg/names/names.go

```diff
@@ -31,3 +36,37 @@ const SERVICE_CA_CONFIGMAP = "openshift-service-ca"
 // MULTUS_VALIDATING_WEBHOOK is the name of the ValidatingWebhookConfiguration for multus-admission-controller
 // that is used in multus admission controller deployment
 const MULTUS_VALIDATING_WEBHOOK = "multus.openshift.io"
+
+// TRUST_BUNDLE_CONFIGMAP_KEY is the name of the data key containing
+// the PEM encoded trust bundle.
+const TRUST_BUNDLE_CONFIGMAP_KEY = "ca-bundle.crt"
+
+// TRUST_BUNDLE_CONFIGMAP is the name of the ConfigMap
+// containing the combined user/system trust bundle.
+const TRUST_BUNDLE_CONFIGMAP = "trusted-ca-bundle"
```

Comment:
s/TRUST_BUNDLE_CONFIGMAP/TRUSTED_CA_BUNDLE_CONFIGMAP/

Reply:
I think we're likely to have multiple different trust bundles.  Let's add proxy to this name so that we can distinguish.  We can always provide combinations at some point, but we cannot resplit.

Reply:
We ended up on this name because the thinking (as driven by @smarterclayton) was that ultimately we would want this to be the mechanism for providing the canonical "CAs to trust" to things on the cluster that wanted to be told what to trust.  e.g. if an admin wanted to entirely replace the system trust bundle, this would eventually be that admin-provided bundle.  (today it aggregates in system, but at some point we might make it optional to not do that).

So given that target use, i'm reluctant to put proxy back in the name of this thing.   But i'm not sure how to reconcile that w/ your goal of "there might be many CA bundles".  

I suppose if we ever reach the world I described above, we could introduce "trusted-ca-bundle" as a thing at that point in time, so I guess i'm ok w/ the proposed rename.

tldr: we started out including proxy in the name, i'm ok going back to it I guess.

But this probably also means you're not going to like the label name that we've socialized for people to put on their configmaps (that gets the CAs injected into the CM).  So we better settle on that asap as we're causing downstream teams churn every time we change direction on it (we already moved them from annotations to labels).

Reply:
@smarterclayton is going to weigh in on this, please don't rename anything yet.